src/sentry/testutils/cases.py

```diff
@@ -1232,6 +1235,62 @@ def _send_buckets(cls, buckets, entity):
         )
 
 
+class BaseMetricsLayerTestCase(ABC, TestCase, BaseMetricsTestCase):
+
+    # The idea of this time is that we want to always have the start of the hour in order to handle
+    # all tests properly. The format can be generalized as follows "****-**-** **:00:00" where "*" stands
+    # for any number in the interval [0-9].
+    DEFAULT_FROZEN_TIME = "2022-09-29 10:00:00"
+
+    @abstractmethod
+    def now(self):
+        """
+        Returns the current time instance that will be used throughout the tests of the metrics layer.
+
+        This method has to be implemented in all the children classes because it serves as a way to standardize
+        access to time.
+        """
+        raise NotImplementedError
+
+    def store_metric_in_metrics_layer(
```

Comment:
Let this take defaults for things like org_id, project_id, ... repetitive things so its actually useful to call without so many params 
Like for example most tests call this function with `self.project_id`

Reply:
Perfect

Reply:
Also this name of the function could use some tweaking maybe, create two methods `store_transaction_metric` and `store_release_health_metric`  for the different use case ids

Reply:
What about `store_performance_metric`?

Reply:
Sure that is good!